src/Microsoft.DotNet.Darc/src/Darc/Operations/SubscriptionStatusesOperation.cs

```diff
@@ -0,0 +1,136 @@
+    {
+        SubscriptionStatusesCommandLineOptions _options;
+
+        public SubscriptionStatusesOperation(SubscriptionStatusesCommandLineOptions options)
+            : base(options)
+        {
+            _options = options;
+        }
+
+        /// <summary>
+        /// Implements the 'subscription-status' operation
+        /// </summary>
+        /// <param name="options"></param>
+        public override async Task<int> ExecuteAsync()
+        {
+            if ((_options.Enable && _options.Disable) ||
+                (!_options.Enable && !_options.Disable))
+            {
+                Console.WriteLine("Please specify either --enable or --disable");
+                return Constants.ErrorCode;
+            }
+
+            string pastTestStatusMessage = _options.Enable ? "enabled" : "disabled";
+            string actionStatusMessage = _options.Enable ? "Enabling" : "Disabling";
+
+            try
+            {
+                IRemote remote = RemoteFactory.GetBarOnlyRemote(_options, Logger);
+
+                bool noConfirm = _options.NoConfirmation;
+                List<Subscription> subscriptionsToEnableDisable = new List<Subscription>();
+
+                if (!string.IsNullOrEmpty(_options.Id))
+                {
+                    // Look up subscription so we can print it later.
+                    try
+                    {
+                        Subscription subscription = await remote.GetSubscriptionAsync(_options.Id);
+                        subscriptionsToEnableDisable.Add(subscription);
+                    }
+                    catch (RestApiException e) when (e.Response.StatusCode == HttpStatusCode.NotFound)
+                    {
+                        Console.WriteLine($"Subscription with id '{_options.Id}' was not found.");
+                        return Constants.ErrorCode;
+                    }
+                }
+                else
+                {
+                    if (!_options.HasAnyFilters())
+                    {
+                        Console.WriteLine($"Please specify one or more filters to select which subscriptions should be {pastTestStatusMessage} (see help).");
+                        return Constants.ErrorCode;
+                    }
+
+                    IEnumerable<Subscription> subscriptions = await _options.FilterSubscriptions(remote);
+
+                    if (subscriptions.Count() == 0)
+                    {
+                        Console.WriteLine("No subscriptions found matching the specified criteria.");
+                        return Constants.ErrorCode;
+                    }
+
+                    subscriptionsToEnableDisable.AddRange(subscriptions);
+                }
+
+                if (!noConfirm)
+                {
+                    // Print out the list of subscriptions about to be enabled/disabled.
+                    Console.WriteLine($"Will {pastTestStatusMessage} the following {subscriptionsToEnableDisable.Count} subscriptions...");
```

Comment:
This will say: 
Will enabled the following x subscriptions.
Looks like this is the only place where we need present tense.